### PR TITLE
Website/get started page

### PR DESCRIPTION
**Pull request by michellewong793**
Get started page minus Knowledge Base

### CODE REVIEW

File: frontend/website-redesign/src/pages/GetStarted.re [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        Theme.desktop: "/static/img/backgrounds/04_GetStarted_1_2880x1504.jpg",
+        Theme.tablet: "/static/img/backgrounds/04_GetStarted_1_1536x1504_tablet.jpg",
+        Theme.mobile: "/static/img/backgrounds/04_GetStarted_1_750x1056_mobile.jpg",
+      }
+    />
+    <ButtonBar
+      kind=ButtonBar.GetStarted
+      backgroundImg="/static/img/ButtonBarBackground.png"
+    />
+    <AlternatingRowsButton
```
Comment (MartinMinkov) [inline frontend/website-redesign/src/pages/GetStarted.re:35]: why was this renamed to append `button`?

Comment (michellewong793) [reply frontend/website-redesign/src/pages/GetStarted.re:35]: the buttons were different than the link, maybe we could have the component accept children instead?

Comment (michellewong793) [reply frontend/website-redesign/src/pages/GetStarted.re:35]: i made a new component bc it was a bit different, didn't have the top numbers either

Comment (MartinMinkov) [reply frontend/website-redesign/src/pages/GetStarted.re:35]: sounds good! was just wondering!
